appcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-slim
+FROM python:3.10
```

Comment:
Asking this just for my own knowledge/understanding: how did you determine that it's not necessary to use the `-slim` variant?

Reply:
Both `-slim` and non`-slim` are built on the same base and both install Python and `pip` and are ready to go for Python code. 

The `-slim` variant just excludes some other tooling like `git`, `curl`, `ssh` etc. that takes up more time/space on the install. `-slim` is a subset of non`-slim`, so we don't lose anything by going up.

The original reason for `-slim` was more about keeping the image size as small as possible and the image contents as minimal as possible. As we've grown the project and had to install new tooling for e.g. supporting different dev platforms (`build-essential` in the base image) -- this has seemed to become less of a concern.

`-slim` felt like an unnecessary restriction that was forcing us to work around in different ways.

Reply:
Oh I see, that makes a lot of sense! Thanks